cephfs/statx.go

```diff
@@ -4,6 +4,10 @@ package cephfs
 #cgo LDFLAGS: -lcephfs
 #cgo CPPFLAGS: -D_FILE_OFFSET_BITS=64
 #include <cephfs/libcephfs.h>
+#ifndef AT_STATX_DONT_SYNC
+// for versions earlier than Quincy
```

Comment:
Backport for the change which introduces `AT_STATX_DONT_SYNC` is already merged with pacific too. So we may need to mention "earlier than Pacific".

Reply:
Wait, what? They removed the define within a released version? 😳

Reply:
Ok. I may have misinterpreted the backport and/or release process. Sorry, the change is not yet in a released version of Pacific and Quincy.

Reply:
Ok, but they merged it on a pacific branch, so it _will_ be released, do I understand that correctly?

Reply:
Yeah, I think that's expected.

Reply:
Crazy. Ok, then I will change of course.